internal/events/webhooks.go

```diff
@@ -114,3 +108,20 @@ func (w *webhookAction) attemptBatch(batchNumber, attempt uint64, events []*even
 	}
 	return err
 }
+
+func (w *webhookAction) validateURL() (*url.URL, *net.IPAddr, error) {
+	u, err := url.Parse(w.spec.URL)
```

Comment:
We know handle this error to late validate if what was provided on the subscription is in fact a URL and not something like a hostname or arbitrary string